`lint` doesn't run on Windows: "Maximum call stack size exceeded"

Upon running the `lint` script, `eslint` failes with `Maximum call stack size exceeded`. This only occurs on Windows.

Node: `v4.2.4`

```
C:\dev\james>npm run lint

> james@1.2.3 lint C:\dev\james
> eslint ./src ./test

C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\esrecurse.js:78
            children = objectKeys(node);
                       ^

RangeError: Maximum call stack size exceeded
    at keys (native)
    at Referencer.Visitor.visitChildren (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\
esrecurse.js:78:24)
    at Referencer.Visitor.visit (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\esrecurs
e.js:112:14)
    at Referencer.Visitor.visitChildren (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\
esrecurse.js:93:26)
    at Referencer.Visitor.visit (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\esrecurs
e.js:112:14)
    at Referencer.Visitor.visitChildren (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\
esrecurse.js:88:38)
    at Referencer.Visitor.visit (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\esrecurs
e.js:112:14)
    at Referencer.Visitor.visitChildren (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\
esrecurse.js:93:26)
    at Referencer.Visitor.visit (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\esrecurs
e.js:112:14)
    at Referencer.Visitor.visitChildren (C:\dev\james\node_modules\eslint\node_modules\escope\node_modules\esrecurse\
esrecurse.js:88:38)

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "
run" "lint"
npm ERR! node v4.2.4
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE
npm ERR! james@1.2.3 lint: `eslint ./src ./test`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the james@1.2.3 lint script 'eslint ./src ./test'.
npm ERR! This is most likely a problem with the james package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint ./src ./test
npm ERR! You can get their info via:
npm ERR!     npm owner ls james
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\dev\james\npm-debug.log
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -74,7 +74,6 @@
     "no-invalid-regexp": 2,          // http://eslint.org/docs/rules/no-invalid-regexp
     "no-irregular-whitespace": 2,    // http://eslint.org/docs/rules/no-irregular-whitespace
     "no-obj-calls": 2,               // http://eslint.org/docs/rules/no-obj-calls
-    "no-reserved-keys": 2,           // http://eslint.org/docs/rules/no-reserved-keys
     "no-sparse-arrays": 2,           // http://eslint.org/docs/rules/no-sparse-arrays
     "no-unreachable": 2,             // http://eslint.org/docs/rules/no-unreachable
     "use-isnan": 2,                  // http://eslint.org/docs/rules/use-isnan
@@ -159,7 +158,7 @@
     "no-new-object": 2,              // http://eslint.org/docs/rules/no-new-object
     "no-spaced-func": 2,             // http://eslint.org/docs/rules/no-spaced-func
     "no-trailing-spaces": 0,         // http://eslint.org/docs/rules/no-trailing-spaces
-    "no-wrap-func": 2,               // http://eslint.org/docs/rules/no-wrap-func
+    "no-extra-parens": 2,            // http://eslint.org/docs/rules/no-extra-parens
     "no-underscore-dangle": 0,       // http://eslint.org/docs/rules/no-underscore-dangle
     "one-var": [2, "never"],         // http://eslint.org/docs/rules/one-var
     "padded-blocks": [2, "never"],   // http://eslint.org/docs/rules/padded-blocks
@@ -173,14 +172,14 @@
     "space-before-function-paren": [2, "never"], // http://eslint.org/docs/rules/space-before-function-paren
     "space-infix-ops": 2,            // http://eslint.org/docs/rules/space-infix-ops
     "space-return-throw-case": 2,    // http://eslint.org/docs/rules/space-return-throw-case
-    "spaced-line-comment": 2,        // http://eslint.org/docs/rules/spaced-line-comment
+    "spaced-comment": 2,        // http://eslint.org/docs/rules/spaced-comment
 
     /**
      * JSX style
      */
+    "jsx-quotes": [2, "prefer-double"],     // http://eslint.org/docs/rules/jsx-quotes
     "react/display-name": 0,         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md
     "react/jsx-boolean-value": 2,    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md
-    "react/jsx-quotes": [2, "double"], // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-quotes.md
     "react/jsx-no-undef": 2,         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-undef.md
     "react/jsx-sort-props": 0,       // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md
     "react/jsx-sort-prop-types": 0,  // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-prop-types.md

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
   "devDependencies": {
     "autoprefixer": "^5.2.0",
     "babel": "^5.5.14",
-    "babel-eslint": "^3.1.23",
+    "babel-eslint": "^5.0.0-beta6",
     "babelify": "^6.1.2",
     "browserify": "^10.2.4",
     "electron-packager": "^5.0.0",

--- a/src/component/requests.js
+++ b/src/component/requests.js
@@ -20,7 +20,7 @@ export default class Requests extends React.Component {
   _onScroll() {
     const scrollableDomNode = React.findDOMNode(this);
     const fromIndex = Math.ceil(scrollableDomNode.scrollTop / requestElementHeight) - 15;
-    this.props.setFromIndex((fromIndex < 0) ? 0 : fromIndex);
+    this.props.setFromIndex(fromIndex < 0 ? 0 : fromIndex);
   }
 
   componentDidMount() {

